### PR TITLE
Add linter and update style

### PR DESCRIPTION
**Pull request by arinaldi**

### CODE REVIEW

File: src/actions/index.js [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
+    description: json.breezometer_description,
   };
-};
+}
 
-export const fetchLocation = location => {
-  return dispatch => {
+export const fetchLocation = (location) => {
+  return (dispatch) => {
     const BOM_API_KEY = 'fbbb89795db54612ad9598e2ce77b709';
```
Comment (jorions) [inline src/actions/index.js]: I would create a dedicated `constants/` folder and put `BOM_API_KEY` in it

File: src/components/HistoryTable.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-//import PropTypes from 'prop-types';
+// import PropTypes from 'prop-types';
```
Comment (jorions) [inline src/components/HistoryTable.js]: Should this just be removed?

File: src/containers/App.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-//import PropTypes from 'prop-types';
+// import PropTypes from 'prop-types';
```
Comment (jorions) [inline src/containers/App.js]: Should this just be removed?

**Review by jorions — CHANGES_REQUESTED** [review]

File: src/components/HistoryTable.js (hunk 2 above)
Comment (arinaldi) [reply src/components/HistoryTable.js]: I uncommented it and added propTypes.

File: src/containers/App.js (hunk 3 above)
Comment (arinaldi) [reply src/containers/App.js]: Done.

File: src/actions/index.js (hunk 1 above)
Comment (arinaldi) [reply src/actions/index.js]: Done.

File: src/actions/index.js [hunk 4]
```diff
@@ -1,25 +1,26 @@
+import { BOM_API_KEY } from '../constants';
+
 export const RECEIVE_LOCATION = 'RECEIVE_LOCATION';
 
 export function receiveLocation(location, json) {
-  //dispatching an action to the store
+  // dispatching an action to the store
```
Comment (ghaynes1292) [inline src/actions/index.js]: One last comment here

**Review by ghaynes1292 — CHANGES_REQUESTED** [review]

**Review by ghaynes1292 — APPROVED** [review]

**Pull request merged**